Formula/shuffledns.rb

```diff
@@ -0,0 +1,19 @@
+class Shuffledns < Formula
+  desc "Enumerate subdomains using active bruteforce & resolve subdomains with wildcards"
+  homepage "https://github.com/projectdiscovery/shuffledns"
+  url "https://github.com/projectdiscovery/shuffledns/archive/refs/tags/v1.0.9.tar.gz"
+  sha256 "ce61eb210c0bb7ff5cc2e0d45e90129764494d9c0b8883e04fe67b16169ab707"
+  license "GPL-3.0-only"
```

Comment:
Seems to be `GPL-3.0-or-later`? https://github.com/projectdiscovery/shuffledns/blob/db34c6634ca891f7a7df9fe75488373212a0130a/LICENSE.md?plain=1#L640